### PR TITLE
Add simple build test on macOS and linux with travis.

### PR DESCRIPTION
**Pull request by abergeron**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+    - os: osx
```
Comment (nouiz) [inline .travis.yml]: In the past, travis didn't allowed a project to test on mac and on linux. Can you confirm that it worked somewhere?

**Comment by nouiz** [comment]
travis didn't run.

File: .travis.yml (hunk 1 above)
Comment (abergeron) [inline .travis.yml]: I tried this PR with my fork on travis and it works.

**Comment by abergeron** [comment]
Of course, we would have to activate it.  But if we do and this isn't merged, we will get a ton spam about failing tests.  I'd rather merge it first and then activate it.

**Comment by nouiz** [comment]
Ok. I merged and activated travis. Here is the first run as I merged
another PR to trigger it:

https://travis-ci.org/Theano/libgpuarray/builds/192416426

**Pull request merged**